src/test/resources/integration/gradle-example/gradle.properties

```diff
@@ -0,0 +1,3 @@
+currentVersion=1.0-SNAPSHOT
+artifactory_user=admin
+artifactory_password=password
```

Comment:
I think the test will fail, when using an Artifactoey with credentials other than admin / password.
Maybe we should inject the values of the configured *JENKINS_ARTIFACTORY_USERNAME* and *JENKINS_ARTIFACTORY_PASSWORD* into this file.